<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetectorVisibility</class>
 <widget class="QWidget" name="DetectorVisibility">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>222</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_Detector">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Toggle System Visibility</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="checkBox_MuonSpectr">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Muon Spectrometer</string>
        </property>
        <property name="shortcut">
         <string>1</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_Magnets">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Magnet System</string>
        </property>
        <property name="shortcut">
         <string>2</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>15</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_Calorimeter">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="title">
         <string>Calorimeters</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <widget class="QCheckBox" name="checkBox_Tilecal">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>34</y>
           <width>145</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Hadronic Calorimeter</string>
         </property>
         <property name="shortcut">
          <string>3</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_LAr">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>61</y>
           <width>145</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>EM Calorimeter</string>
         </property>
         <property name="shortcut">
          <string>4</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_ID">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>150</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="title">
         <string>Inner Detector</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <widget class="QCheckBox" name="checkBox_SCT">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>61</y>
           <width>60</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>SCT</string>
         </property>
         <property name="shortcut">
          <string>6</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_TRT">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>34</y>
           <width>60</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>TRT</string>
         </property>
         <property name="shortcut">
          <string>5</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_Pixels">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>88</y>
           <width>60</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Pixels</string>
         </property>
         <property name="shortcut">
          <string>7</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
